src/AspNet.Security.OAuth.Moodle/MoodleAuthenticationConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.Moodle
+{
+    /// <summary>
+    ///     Contains constants specific to the <see cref="MoodleAuthenticationHandler" />.
```

Comment:
Why did you change the indentation of all the XML comments like this to be different to all the code in the rest of the code base?
```suggestion
    /// Contains constants specific to the <see cref="MoodleAuthenticationHandler" />.
```

Reply:
Maybe made by ReSharper....